### PR TITLE
Fixes #3: GET /agencies/:id

### LINKED ISSUE #3: Create GET /agencies/:id

### PR DESCRIPTION
**Pull request by jhannah**

### CODE REVIEW

File: src/agency/agency.service.ts [hunk 1]
```diff
@@ -9,6 +9,14 @@ export class AgencyService {
     @InjectRepository(Agency) private agencyRepository: Repository<Agency>,
   ) {}
 
+  findAll(): Promise<Agency[]> {
+    return this.agencyRepository.find();
+  }
```
Comment (dodgees) [inline src/agency/agency.service.ts]: I don't think findAll is used. Can be deleted

**Comment by dodgees** [comment]
Where are the tests, Jay!!!

**Review by dodgees — APPROVED** [review]

**Comment by jhannah** [comment]
What is this? It's not an error? But it is? But all tests passed?
```
 PASS  src/agency/agency.e2e-spec.ts
  ● Console

    console.error
      Unhandled error

      at process.uncaught (../node_modules/jest-jasmine2/build/jasmine/Env.js:248:21)

    console.error
      QueryFailedError: Connection terminated
          at new QueryFailedError (/Users/jhannah/src/codefornebraska/public-salaries-api/src/error/QueryFailedError.ts:11:9)
          at PostgresQueryRunner.<anonymous> (/Users/jhannah/src/codefornebraska/public-salaries-api/src/driver/postgres/PostgresQueryRunner.ts:228:19)
          at step (/Users/jhannah/src/codefornebraska/public-salaries-api/node_modules/tslib/tslib.js:143:27)
          at Object.throw (/Users/jhannah/src/codefornebraska/public-salaries-api/node_modules/tslib/tslib.js:124:57)
          at rejected (/Users/jhannah/src/codefornebraska/public-salaries-api/node_modules/tslib/tslib.js:115:69)
          at processTicksAndRejections (node:internal/process/task_queues:96:5)

      at process.uncaught (../node_modules/jest-jasmine2/build/jasmine/Env.js:249:21)

    console.log
      Agencies already created

      at BootstrapDataService.insertAgencyData (bootstrap/bootstrap-data.service.ts:67:15)

    console.log
      Finished bootstrapping agency data.

      at bootstrap/bootstrap-data.service.ts:21:15
 ```

**Comment by jhannah** [comment]
Hmm I'm also getting this:
```
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "QueryFailedError: Connection terminated".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
```

**Pull request merged**